cmd/generate/main.go

```diff
@@ -19,6 +19,8 @@ func main() {
 	var (
 		sourceCodePath = flag.String("src-path", "",
 			"The path of source code your want to generate support from, it can be either a file or a directory. (required)")
+		directiveMapName  = flag.String("directive-map", "directives", `Name of the generated map variable. "Directives" by default. (optional)`)
+		matchFuncName     = flag.String("match-func", "Match", `Name of the generated matchFunc. "Match" by default. (optional)`)
```

Comment:
If these _need_ to be provided (as the shell script appears to enforce) then make them required here and cmd/generate now works better when invoked outside the shell script

Reply:
My origin thought was: `public_git.sh` only targets for updating the `.gen.go` files in crossplane so it is required(these two arguments should follow the pattern in other `.gen.go`). `./cmd/generate` may be called by some users just want to generate a temporary support for a third-party module, so I provide a default value just for their convenience. 
But I'm unsure if this difference(`required` in `public_git.sh`, `optional` in `./cmd/generate`) will cause maintainers feel confused. Appreciate if you can provide some insights here.